pyrene/src/components/DateTimePicker/DateTimePicker.tsx

```diff
@@ -0,0 +1,303 @@
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+
+import ReactDatePickerWrapper from './ReactDatePickerWrapper/ReactDatePickerWrapper';
+import DateTimeInput from './DateTimeInput/DateTimeInput';
+
+import {
+  getFutureDate, customDateFormat, convertToUTCtime,
+  convertToZoneTime, customStringToDate, getErrors,
+  hasDateError, hasTimeError, getClientTimeZone,
+} from '../../utils/DateUtils';
+
+type OnFunction = (value?: number) => void;
+
+export interface DateTimePickerProps{
+  /**
+   * Calendar is opened on input component render
+   */
+  calendarOpened?: boolean,
+  /**
+   * Close calendar on date select
+   */
+  closeOnSelect?: boolean,
+  /**
+   * Date format used by component
+   */
+  dateFormat?: string,
+  /**
+   * Boolean to toggle time display
+   */
+  dateOnly?: boolean,
+  /**
+   * Date object that represents the end date of the component
+   */
+  endDate?: Date,
+  /**
+   * Display the component inline
+   */
+  inline?: boolean,
+  /**
+   * String label of the component
+   */
+  label?: string,
+  /**
+   * Unix timestamp that represents the maximum date allowed by the component
+   */
+  maxDateTime?: number,
+  /**
+   * Unix timestamp that represents the minimum date allowed by the component
+   */
+  minDateTime?: number,
+  /**
+   * Name that can be used to uniquely identify the component
+   */
+  name?: string,
+  /**
+   * Calendar opened callback
+   */
+  onCalendarOpen?: () => void,
+  /**
+   * Click outside componet callback
+   */
+  onClickOutside?(event: React.MouseEvent<HTMLDivElement>): void,
```

Comment:
I think we should type all functions in the same way, so use arrows in this case (do it the same as you did `onCalendarOpen` a few lines above):
`onClickOutside?: (event: React.MouseEvent<HTMLDivElement>) => void,`

Reply:
✅